code/api/src/migrations/9-deliveries.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.renameTable('productsHistory', 'deliveries')
```

Comment:
This name makes sense to me. I think user_product would have also been descriptive.